cuebot/src/main/resources/conf/ddl/postgres/migrations/V34__Add_loki_job_fields.sql

```diff
@@ -0,0 +1,6 @@
+alter table job
+    add b_loki_enabled bool;
+
+alter table job
+    add str_loki_url varchar(256);
```

Comment:
What if we make this a single field, `str_loki_url`? If not `Null,` Loki is enabled.
This will avoid the ambiguous states where enable + empty and disable + non-empty.

The same goes for the values defined at DispatchFrame

Reply:
I was thinking much the same thing yesterday 🙂
I think the bool was from the previous implementation.

I'll have a look at changing it to just use the string.

Reply:
This has been implemented